tutorials/plot_forward.py

```diff
@@ -86,15 +86,26 @@
                        surfaces='head-dense')
 
 ###############################################################################
+# .. _plot_forward_source_space:
+#
 # Compute Source Space
 # --------------------
 #
-# The source space defines the position of the candidate source locations.
-# The following code compute such a cortical source space with
-# an OCT-6 resolution.
+# The source space defines the position and orientation of the candidate source
```

Comment:
from the original doc in `manual/forward.rst`, I understand that the source space also contains orientations. But based on what `cookbook.rst` says I'm not sure of it. 

If the source space has orientation information shouldn't it be also cross referenced to the dipole orientation of the inverse solution?

Reply:
Yes the source space contains the definition of the normals / directions for the source points.

The inverse solution can choose whether to use only the normal orientation (fixed) or all three directions (free) for a given source space location, or somewhere in between (loose). I hope that answers your question.